src/components/delegateList/votingRow.js

```diff
@@ -11,7 +11,7 @@ const setRowClass = (voteStatus) => {
   if (pending) {
     return styles.pendingRow;
   } else if (confirmed !== unconfirmed) {
-    return confirmed ? styles.downVoteRow : styles.upVoteRow;
+    return confirmed ? `${styles.downVoteRow} votedRow` : `${styles.upVoteRow} votedRow`;
```

Comment:
We use dash-separated class names for tests - `voted-row`. 
But the class should IMO be named `vote-row` and be set directly on the `ul` in https://github.com/LiskHQ/lisk-hub/blob/bc21d45ef1456e57edde098aa19f2e2c52813ca1/src/components/delegateList/votingRow.js#L32

Reply:
It is possible but I will need to write same conditions directly on `ul` because I need that class specifically for rows that were selected, otherwise I would use `delegate-row`.
I can't use `delegate-row` to count it because in HTML it shows all rows and hides on that shouldn't be displayed.